bedrock/firefox/templates/firefox/new/sem/base.html

```diff
@@ -2,15 +2,17 @@
  # License, v. 2.0. If a copy of the MPL was not distributed with this
  # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
 
+{% add_lang_files "firefox/new/sem" %}
```

Comment:
Do you think it's worth a comment along with this lang file import detailing the limited localization? Might be good to know it's only `en-US` and `de` at a quick glance for when we forget the details next week. 😝